slices/index.ts

```diff
@@ -1,29 +1,30 @@
 // Code generated by Slice Machine. DO NOT EDIT.
 
-import { defineAsyncComponent } from 'vue'
-import { defineSliceZoneComponents } from '@prismicio/vue'
+import { defineAsyncComponent } from "vue";
```

Comment:
I think you might have an auto format setting enable that is changing single quotes into double quotes. When I run `npm run dev`, I'm getting lint errors related to this log in the console.